create-table-as.md

```diff
@@ -1,18 +1,34 @@
 ---
 title: CREATE TABLE AS
-summary: The CREATE TABLE AS statement creates a new table from the results of a SELECT statement.
+summary: The CREATE TABLE AS statement persists the result of a query into the database for later reuse.
 toc: false
 ---
 
-The `CREATE TABLE AS` statement creates a new table from the results of a [`SELECT`](select.html) statement.
-
-These tables are not designed for long-term use, as they do not support some common table features like [Primary Keys](primary-key.html) and [interleaving](interleave-in-parent.html). For similar functionality with more robust feature support, [create a table](create-table.html) and then use [`INSERT INTO SELECT`](insert.html#insert-from-a-select-statement).
+The `CREATE TABLE ... AS` statement creates a new table from the results of a query.
 
 <div id="toc"></div>
 
+## Intended use
```

Comment:
Nice! Just a few nits:

- Use title case for the header: `Intended Use`
- Make each bullet a sentence, that is, capitalize `The` and use a period instead of a semicolon.